bom/runtime/pom.xml

```diff
@@ -2166,6 +2166,10 @@
                         <groupId>com.fasterxml.jackson.core</groupId>
                         <artifactId>jackson-core</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>com.fasterxml.jackson.module</groupId>
+                        <artifactId>jackson-module-scala_2.12</artifactId>
```

Comment:
It's unfortunate that the version appears in the artifactId. I'm wondering whether it'll be a better maintenance compromise to exclude everything under `com.fasterxml.jackson.module` in this case. At this point this is the only artifact from this groupId that appears as a dependency of `io.vertx:vertx-kafka-client` through `org.apache.kafka:kafka_2.12`.
Unfortunately, Maven does not support exclusion expressions, it's either `*` or specific artifactId.

Reply:
```
[INFO] +- org.apache.kafka:kafka_2.12:jar:2.3.0:compile
[INFO] |  +- com.fasterxml.jackson.module:jackson-module-scala_2.12:jar:2.9.9:compile
[INFO] |  |  \- com.fasterxml.jackson.module:jackson-module-paranamer:jar:2.9.9:compile
```

Reply:
I have changed it to a * exclude

Reply:
Looks like the * approach does not work, I think it must only work to exclude everything.